roles/generate_kubernetes_config/templates/dugtrio.yaml.j2

```diff
@@ -5,6 +5,7 @@ dugtrio:
   image:
     repository: {{ default_tooling_images.dugtrio.split(':') | first }}
     tag: {{ default_tooling_images.dugtrio.split(':') | last }}
+    pullPolicy: Always
```

Comment:
why did you add this again?

Reply:
Here we can have this pull policy, just not in the upstream ethereum helm chart.

Reply:
This would roll the updates on devnets

Reply:
tbh, I'd leave it with IfNotPresent here.
It's actually the same as with the upstream chart - the way to roll image is providing a new tag, so we do not accidentally introduce new bugs just by restarting the pod.
Default tag name is now "1.0.0", which doesn't change anyway

Reply:
can we make the pullPolicy conditional?
So use `Always` if `default_tooling_images.dugtrio.split(':') | last ` is `latest`,  if not use `IfNotPresent`

Reply:
I don't wanna be too picky on that, so I'm gonna approve it now ;)
However, it doesn't really make sense to set it to `IfNotPresent` in the upstream helm chart, just to override it with `Always` in literally all devnets :D
It's then basically the same as if we just set `Always` in the upstream repo..

Reply:
Just noticed the change, 
Cool 👍

Reply:
assuming that we are the only ones using the helm chart, I agree, but the general idea is that there might be others that use the helm chart, and we wanna make sure its gonna work for them too.